ckanext/subscribe/notification.py

```diff
@@ -301,7 +301,8 @@ def get_notifications_by_email(activities, objects_subscribed_to,
             # ignore activity that occurs before this subscription was created
             if subscription.created > activity.timestamp:
                 continue
-
+            if not subscription.verified:
+                continue
```

Comment:
I can't believe subscription verification wasn't checked before - it really looks like it didn't do anything at all. :(

I think it would be cleaner, however, if we add a filter for verification==True to the database queries in `get_objects_subscribed_to`: https://github.com/opendata-swiss/ckanext-subscribe/blob/master/ckanext/subscribe/notification.py#L137. Do you think you could change this PR to do that instead?